pkg/apis/admission/register.go

```diff
@@ -27,16 +27,6 @@ const GroupName = "admission.k8s.io"
 // SchemeGroupVersion is group version used to register these objects
 var SchemeGroupVersion = schema.GroupVersion{Group: GroupName, Version: runtime.APIVersionInternal}
 
-// Kind takes an unqualified kind and returns a Group qualified GroupKind
-func Kind(kind string) schema.GroupKind {
```

Comment:
These functions represent a sort of API to the api packages.  It sure seems like this is not used, but I don't know why it was there.

@lavalamp or @deads2k or @jpbetz - can we get a consult?

Reply:
I would not remove these unless you can show that no one uses the corresponding function from any api package.

Reply:
As discussed in comments - we should leave these ones.  They represent a package-api.